### ISSUE TITLE
Hashie is back

### ISSUE THREAD

**Issue by gonmf** [shideneyu/kraken_client#16]
Hello, thank you for your good work. 

When I try using version 1.2 I get:
```
gems/kraken_client-1.2/lib/kraken_client/application.rb:7:in `require': cannot load such file -- hashie (LoadError)
	from /usr/local/lib/ruby/gems/2.4.0/gems/kraken_client-1.2/lib/kraken_client/application.rb:7:in `<top (required)>'
	from /usr/local/lib/ruby/gems/2.4.0/gems/kraken_client-1.2/lib/kraken_client.rb:4:in `require_relative'
	from /usr/local/lib/ruby/gems/2.4.0/gems/kraken_client-1.2/lib/kraken_client.rb:4:in `block in <top (required)>'
	from /usr/local/lib/ruby/gems/2.4.0/gems/kraken_client-1.2/lib/kraken_client.rb:3:in `each'
	from /usr/local/lib/ruby/gems/2.4.0/gems/kraken_client-1.2/lib/kraken_client.rb:3:in `<top (required)>'
....
```

**Comment by shideneyu** [comment]
Hello, thank you for your kind message.

I've just fixed it in version ´1.2.1´ , it's my fault for solely relying in the specs and not in actual testing. :+1: 

Please not that Hashie is not used anymore, due to Kraken sometime returning the key 'count' in their reponses, which might ensue instabilities with Hashie since `count` is a reserved method name,  instabilities that we cannot tolerate for a trading tool :)

**Comment by shideneyu** [comment]
https://github.com/shideneyu/kraken_client/commit/94d7729e77dbf7d29045a0360d7f6dbd33b9696b
